compiler/src/dmd/file_manager.d

```diff
@@ -139,7 +234,6 @@ nothrow:
                 }
                 FileName.free(n2.ptr);
             }
-            FileName.free(n.ptr);
```

Comment:
Note, I moved this to a scope(exit) on line 217 because it was a memory leak if the package.d file was found.